lib/core/presentation/widgets/common/photos_gallery_view/photo_gallery_view.dart

```diff
@@ -0,0 +1,122 @@
+import 'package:app/theme/color.dart';
+import 'package:app/theme/spacing.dart';
+import 'package:app/theme/typo.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_screenutil/flutter_screenutil.dart';
+import 'package:smooth_page_indicator/smooth_page_indicator.dart';
+
+final IMAGE_SIZE = 351.w;
```

Comment:
this is the fullwidth of the device, consider using full width rather than using fixed number like this

Reply:
you can also consider using AspectRatio 
https://www.youtube.com/watch?v=XcnP3_mO_Ms